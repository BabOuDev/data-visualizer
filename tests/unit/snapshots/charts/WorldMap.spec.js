import {shallowMount} from '@vue/test-utils';
import WorldMap from '@/components/charts/WorldMap.vue';

describe('WorldMap.vue', () => {
  it('should renders correctly', () => {
    const component = shallowMount(WorldMap, {
      props: {
        rows: [
          {contact: {location: {lon: -180, lat: -90}}},
          {contact: {location: {lon: -180, lat: 0}}},
          {contact: {location: {lon: -180, lat: 90}}},
          {contact: {location: {lon: 0, lat: -90}}},
          {contact: {location: {lon: 0, lat: 0}}},
          {contact: {location: {lon: 0, lat: 90}}},
          {contact: {location: {lon: 180, lat: -90}}},
          {contact: {location: {lon: 180, lat: 0}}},
          {contact: {location: {lon: 180, lat: 90}}},
        ],
      },
    });
    expect(component.html()).toMatchSnapshot();
  });
});
