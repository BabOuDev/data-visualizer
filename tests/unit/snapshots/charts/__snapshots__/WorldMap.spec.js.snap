// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WorldMap.vue should renders correctly 1`] = `
<div class="map-container background">
  <div class="map"></div>
  <div style="left: 0px; top: 412px;" class="pin" title="undefined undefined"></div>
  <div style="left: 0px; top: 206px;" class="pin" title="undefined undefined"></div>
  <div style="left: 0px; top: 0px;" class="pin" title="undefined undefined"></div>
  <div style="left: 400px; top: 412px;" class="pin" title="undefined undefined"></div>
  <div style="left: 400px; top: 206px;" class="pin" title="undefined undefined"></div>
  <div style="left: 400px; top: 0px;" class="pin" title="undefined undefined"></div>
  <div style="left: 800px; top: 412px;" class="pin" title="undefined undefined"></div>
  <div style="left: 800px; top: 206px;" class="pin" title="undefined undefined"></div>
  <div style="left: 800px; top: 0px;" class="pin" title="undefined undefined"></div>
  <!--v-if-->
</div>
`;
